modular_citadel/code/modules/mob/living/carbon/human/species_types/furrypeople.dm

```diff
@@ -0,0 +1,2 @@
+/datum/species/aquatic
```

Comment:
You don't need to edit this, because furrypeople.dm is our snowflake file.

Reply:
Could probably move the entire furrypeople.dm to modular_citadel for the sake of consistency

Reply:
So move furrypeople.dm to modular_citadel?